src/assets/scripts/client/airport/AirportWind.js

```diff
@@ -0,0 +1,35 @@
+import AirportModel from './AirportModel';
+import { calculateNextWind } from '../utilities/windUtilities';
+
+/**
+ * Mutable array that holds the information on the airport's current wind.
+ */
+export let airportWind = {
+   speed: 10,
```

Comment:
these two properties can be made class properties of the `AirportWindModel`

Reply:
@n8rzz Would it be prudent to set up some sort of broad-scope weather class? Like an `AtmosphereModel` or something? That could have wind/pressure/etc properties and methods all in a single place. just a thought.

Reply:
Maybe someday. It I feel that would be an over abstraction at the moment.